libnavui-androidauto/src/main/java/com/mapbox/androidauto/MapboxCarApp.kt

```diff
@@ -50,5 +50,6 @@ object MapboxCarApp {
     fun setup(application: Application) {
         SharedApp.setup(application)
         MapboxNavigationApp.registerObserver(CarAppLocationImpl())
+        System.setProperty("com.mapbox.mapboxsearch.enableSBS", true.toString())
```

Comment:
Have we considered bumping to the latest Search SDK version available [`1.0.0-beta.34`](https://github.com/mapbox/mapbox-search-android/releases/tag/v1.0.0-beta.34)?

If so, we should remove `System.setProperty("com.mapbox.mapboxsearch.enableSBS", "true")` and use a new way to create a `SearchEngine`.

cc @Zayankovsky @DzmitryFomchyn

Reply:
Oh this can be removed? done!

Reply:
Only after bumping into [`1.0.0-beta.34`](https://github.com/mapbox/mapbox-search-android/releases/tag/v1.0.0-beta.34) 😛